Part_1/Exercise/config.py

```diff
@@ -1 +1 @@
-TOKEN_API="6599937734:AAE-x2cyalUMq44q2aLzy9B2AoZr0_hG2A"
\ No newline at end of file
+TOKEN_API=""
```

Comment:
this should not be changed in this exercise folder. But you can change it in your own /Liban folder